bridge/service/docker.py

```diff
@@ -60,10 +62,16 @@ def start(self):
         self.pull_image()
         self.start_container()
         self.ensure_ready()
+        self.register()
         console.print(
             f"[bold bright_green]Service [white]{self.config.name}[/white] started!"
         )
 
+    def register(self):
+        bridge_cid_path = resolve_dot_bridge() / "cid"
+        with open(bridge_cid_path, "a") as f:
+            f.write(f"{self.container_id}\n")
```

Comment:
Not super urgent since they'll be no-ops in 'stop', but I think this will result in duplicates until someone runs 'stop'.